src/ansys/fluent/core/services/settings.py

```diff
@@ -274,6 +274,11 @@ def _extract_static_info(self, info: SettingsModule.StaticInfo) -> dict[str, Any
             ret["object-type"] = self._extract_static_info(info.object_type)
         if info.help:
             ret["help"] = info.help
+        try:
+            if info.file_purpose:
+                ret["file_purpose"] = info.file_purpose
```

Comment:
@hpohekar Looking above, I note that existing keys contain "-" and not "_".

Reply:
Right, it should be "file-purpose". Thank you.

Reply:
It's replaced with `attrs` now.